src/version/delete-version.ts

```diff
@@ -58,13 +58,13 @@ export function deletePackageVersions(
     deletePackageVersion(id, token).pipe(
```

Comment:
Loop over the found version IDs and pass them to the API to be deleted. This file is mostly taken from the official github action to [delete oldest packages](https://github.com/marketplace/actions/delete-package-versions), with the exception of me adding a `dryRun` mode that just prints the versions to be deleted and exists without deleting.